.github/workflows/docker.yml

```diff
@@ -6,11 +6,7 @@ name: Docker build
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    tags: [ v*.*.* ]
```

Comment:
This is more of a question about docker: Does this `tags` entry mean, that only pushes that create tags trigger this build, or is it additional?

Reply:
This line means that any tag using a v*.*.* form (following a globing expansion method) will trigger the docker workflow on github actions. I changed this line to include the tags build_*. We may add other forms if your work on the CI requires it ofc.

We don't get builds for any push (for now). This was intended because I thought it would be overkill. Keep in mind that I try to have the docker images "production ready" (it's a best effort, since eradiate itself is still under dev). So the docker build may be a bit slow to do it in preamble of a CI that you want to trigger at each push. We could reuse parts of the docker work to create a cheaper CI executor.

Another solution is to schedule daily builds and CI runs. This depends on the workflow you want to implement with the CI. We can also think of other triggers that could make sense in the CI. For instance, a build that occurs on any push on specific branches? Use it mostly for PR? We could have a testing branch closely following the main branch, or directly use the main or next branch for this purpose? This really depends on what you want to achieve with the CI. What do you think also @leroyvn ?

I suggest to discuss this on a CI related PR. This one is more focused on the Docker build itself, which happen to be integrated in github actions